codegen-sbt/src/sbt-test/codegen/test-compile-option/build.sbt

```diff
@@ -0,0 +1,24 @@
+import _root_.caliban.tools.Codegen
```

Comment:
I think you can simply add the test case to the existing schema, to keep things simpler 😄

Reply:
👍 I moved in into the existing compile test